src/Umbraco.Web.UI.Client/src/views/media/media.edit.controller.js

```diff
@@ -194,7 +194,7 @@ function mediaEditController($scope, $routeParams, $q, appState, mediaResource,
 
                     contentEditingHelper.handleSaveError({
                         err: err,
-                        redirectOnError: !infiniteMode,
+                        redirectOnFailure: !infiniteMode,
```

Comment:
Are there other places in the code that have the incorrect property name still in use for `handleSaveError` @bergmania

Reply:
Nope